Basic-Car-Maintenance/Documentation.docc/FirestoreDetails.md

```diff
@@ -0,0 +1,79 @@
+| actionText | Text           | Indicates the text that will appear on the action button                               |
+| actionURL  | Text           | Indicates the url that the user will be redirected to when the acton button is pressed |
+| emojiIcon  | Text           | The emoji that will appear on the alert message                                        |
+| id         | Text           | Identifies a specific alert                                                            |
+| isOn       | Boolean        | Indicates if the alert is active                                                       |
+| message    | Text           | The text content of the alert                                                          |
+| title      | Text           | The name given to the alert                                                            |
+
+### _maintainance_events_
+
+Maintainance events collection contains the individual maintainance events that are associated
+with a specific user.
+
+| Fields    | Firestore Type | Description                                                                               |
+| --------- | -------------- | ----------------------------------------------------------------------------------------- |
+| id        | DocumentID     | The unique maintainance event identifier that will auto-populate your objects id property |
+| userID    | Text           | Identifies the user associated with the maintainance event                                |
+| title     | Text           | The customized name given to the maintainance event                                       |
+| date      | Date           | The date that the maintainance event was created                                          |
+| notes     | Text           | Extra details that the user want to provide about the maintainance event                  |
+| vehicleID | Text           | Identifies the vehicle assocated with the maintainance event                              |
+
+### _vehicles_
+
+The vehicles collection contains all the vehicles associated with a specific user.
+
+| Fields | Firestore Type | Description                                                                    |
+| ------ | -------------- | ------------------------------------------------------------------------------ |
+| id     | DocumentID     | The unique vehicle identifier that will auto-populate your objects id property |
+| userID | Text           | Identifies the user associated with the vehicle                                |
+
+| name | Text | Customized name given to vehicle entry
+| make | Text | Indicates the brand of the vehicle
+| model | Text | Indicates unique name associated with a specific brand
+| year | Text | Indicates the year the vehicle was released by the manufacturer
+| color | Text | Indicates the manifacturer color associated with the vehicle
```

Comment:
Instead of using the word "indicates" in front of a lot of properties, you can leave the word off (can you change all the properties this affects